### PR TITLE
Modernize

### PR DESCRIPTION
**Pull request by Roman3349**
- Add support for PHP 8.0
- Update development dependencies
- Use GitHub Actions instead of Travis CI
- Modernize readme

### CODE REVIEW

**Comment by vody105** [comment]
@f3l1x are we including this in v0.7?

File: src/Schema/Validation/NegotiationValidation.php [hunk 1]
```diff
@@ -40,26 +38,30 @@ public function validate(SchemaBuilder $builder): void
...
-						if (!class_exists($renderer)) {
-							throw new InvalidSchemaException(sprintf(
-								'Negotiation renderer "%s" in "%s::%s()" does not exists',
-								$renderer,
-								$controller->getClass(),
-								$method->getName()
-							));
-						}
+					if ($renderer === null) {
+						continue;
```
Comment (f3l1x) [inline src/Schema/Validation/NegotiationValidation.php:45]: Please keep original construction.

File: composer.json [hunk 2]
```diff
@@ -34,17 +34,18 @@
...
+    "phpstan/phpstan-deprecation-rules": "^0.12",
+    "phpstan/phpstan-nette": "^0.12",
+    "phpstan/phpstan-strict-rules": "^0.12",
     "psr/log": "^1.1",
     "symfony/validator": "^4.1.1|^5.0.0"
   },
   "conflict": {
-    "nette/di": "<=3.0.0-RC"
+    "nette/di": "<=3.0.0-RC",
+    "guzzlehttp/psr7": "<1.7.0"
```
Comment (vody105) [inline composer.json:48]: There is no need for this now because we require contributte/psr7 0.7+ and it requires psr >1.7.0

File: src/Annotation/Controller/Path.php [hunk 3]
```diff
@@ -20,7 +20,7 @@ final class Path
 	 */
 	public function __construct(array $values)
 	{
-		if (!isset($values['value'])) {
+		if (!array_key_exists('value', $values)) {
```
Comment (vody105) [inline src/Annotation/Controller/Path.php:23]: @Roman3349 why did you change this?

**Comment by f3l1x** [comment]
Merged via https://github.com/apitte/core/commit/462e37e17e9c0ff783fc482859b49537b84eb7d5, thanks.
